### ISSUE TITLE
add support for windows

### ISSUE THREAD

**Issue by MalteLond** [lorenzha/hdcd#28]
I can't get foreach to work on a windows. I am currently installing windows but obviously it would be interesting to give parallel processing also to users with a windows machine

**Comment by lorenzha** [comment]
I agree. Haven't used it on windows yet :) I would prefer to first "finish" it on one system and then see what it takes to port it at least to Windows. But if you want to tackle that at the same time be my guest.

**Comment by MalteLond** [comment]
OK. Then I'll just use linux in the future and care about windows when this becomes relevant.
